app/Http/Controllers/ChirpController.php

```diff
@@ -89,4 +90,16 @@ public function destroy(Chirp $chirp): RedirectResponse
 
 	return redirect(route('chirps.index'));
     }
+
+    /**
+     * display the chirps from only the past week
+     */
+
+     public function chirpsFromPastWeek()
+     {
+        $weekAgo = Carbon::now()->subWeek();
```

Comment:
change this to a more general method named recentChirps($daysPast) which takes in a number of days and returns the view with chirps from that amount of days past. the default value should be 14.

Reply:
Got it changed